.travis.yml

```diff
@@ -11,7 +11,6 @@ install:
 before_script:
 - bin/install_client_deps.sh
 script:
-- bash -c '! lesscpy -N static/less/base.less 2>&1 | grep Error'
```

Comment:
Note: this was needed because lesscpy errors don't bubble up to the django renderer. This is no longer needed as scss errors will come up in the template rendering exception (and thus be caught in response tests)